app/controllers/admin/poll/questions_controller.rb

```diff
@@ -55,11 +56,21 @@ def destroy
   private
 
     def question_params
-      params.require(:poll_question).permit(:poll_id, :title, :question, :proposal_id)
+      attributes = [:poll_id, :title, :question, :proposal_id]
+      params.require(:poll_question).permit(*attributes, *translation_params(params[:poll_question]))
     end
 
     def search_params
       params.permit(:poll_id, :search)
     end
 
+    def resource
+      @poll_question = Poll::Question.find(params[:id]) unless @poll_question
+      @poll_question
```

Comment:
Is this line necessary? As far as I know, Ruby always returns the value of the last statement but if you can provide some insight @microweb10, that'd be great :smile_cat:

Reply:
Resolved as per [this comment](https://github.com/consul/consul/pull/2833#discussion_r215309891)

Reply:
@microweb10 As per commit 27b90cb, I think you could apply the same patch in here :ok_hand:

Reply:
👍 https://github.com/consul/consul/pull/2833/commits/7373a41a2868c982c3caea924378091d4052b890